modules/govuk/manifests/node/s_base.pp

```diff
@@ -15,7 +15,7 @@
   include backup::client
   include base
   # Remove conditional to install filebeat on other environments
-  if $::environment =~ /vagrant|integration/ {
+  if $::domain =~ /^(dev|.*\.integration\.publishing\.service)\.gov\.uk/ {
```

Comment:
I think this needs to be:
`^(.*\.dev|.*\.integration\.publishing\.service)\.gov\.uk`

Otherwise it won't match a dev domain such as "ci.dev.gov.uk" because the bit before "dev" isn't being matched.